src/main/java/bio/terra/cda/app/controller/QueryApiController.java

```diff
@@ -18,58 +20,79 @@
 @Controller
 public class QueryApiController implements QueryApi {
 
-  public static final int DEFAULT_LIMIT = 100;
-  public static final int DEFAULT_OFFSET = 0;
+  public static final int DEFAULT_PAGE_SIZE = 100;
+  public static final int DEFAULT_OFFSET = 100;
```

Comment:
It seems kinda weird to start at 100 no?

Reply:
It should be `0`. 🤦

Reply:
Does DEFAULT_OFFSET still need to be reverted to 0

Reply:
I ended up deleting this code as it was unnecessary. These values are always set by the time they reach the controller code, so there's no need to have defaults here as well as in the openapi yaml.